Classes/Sitegeist/MagicWand/Command/CloneCommandController.php

```diff
@@ -372,15 +385,19 @@ public function remoteHostCommand(
     protected function checkConfiguration($remotePersistenceConfiguration)
     {
         $this->outputHeadLine('Check Configuration');
-        if ($remotePersistenceConfiguration['driver'] != 'pdo_mysql'
-            && $this->databaseConfiguration['driver'] != 'pdo_mysql') {
-            $this->outputLine(sprintf('<error>Error:</error>  Only mysql is supported. Remote: "%s" Local: "%s" configured.', $remotePersistenceConfiguration['driver'], $this->databaseConfiguration['driver']));
+        if (!$this->dbal->driverIsSupported($remotePersistenceConfiguration['driver'])
+            && !$this->dbal->driverIsSupported($this->databaseConfiguration['driver'])) {
+            $this->outputLine(sprintf('<error>ERROR:</error> Only pdo_pgsql and pdo_mysql drivers are supported! Remote: "%s" Local: "%s" configured.', $remotePersistenceConfiguration['driver'], $this->databaseConfiguration['driver']));
+            $this->quit(1);
+        }
+        if ($remotePersistenceConfiguration['driver'] !== $this->databaseConfiguration['driver']) {
+            $this->outputLine('<error>ERROR:</error> Remote and local databases must use the same driver!');
```

Comment:
It would be cool to output which one is configured locally and which one is used on the remote server.

Reply:
Fixed.